v2/kafka-to-bigquery/src/main/java/com/google/cloud/teleport/v2/templates/KafkaToBigQueryFlex.java

```diff
@@ -268,7 +287,9 @@ public static PipelineResult runAvroPipeline(
       }
 
       if (options.getSchemaRegistryConnectionUrl() != null && options.getOutputDataset() != null) {
-        writeResult = kafkaRecords.apply(AvroDynamicTransform.of(options));
+        writeResult =
+            kafkaRecords.apply(
+                AvroDynamicTransform.of(options).withBadRecordErrorHanlder(badRecordErrorHandlers));
```

Comment:
It looks like currently DLQ is only added in the case when we use BQ Dynamic Destinations. Can you also add the DLQ to AvroTransform which covers the other paths.

Reply:
Done pending testing